File: Code 3 Callouts/Code 3 Callouts/Models/Vehicles/VehicleHelper.cs
using Rage;
using Stealth.Common.Models;
using Stealth.Common.Natives;

namespace Stealth.Plugins.Code3Callouts.Models.Vehicles
{

	internal static class VehicleHelper
	{

		internal static VehicleColor GetVehicleColors(Rage.Vehicle v)
		{
			return Stealth.Common.Natives.Vehicles.GetVehicleColors(v);
		}

		internal static void SetVehicleColors(Rage.Vehicle v, int pPrimColor, int pSecColor)
		{
            Stealth.Common.Natives.Vehicles.SetVehicleColors(v,
                new VehicleColor((Stealth.Common.Natives.Vehicles.EPaint)pPrimColor, (Stealth.Common.Natives.Vehicles.EPaint)pSecColor)
                );
		}

	}

}

File: Code 3 Callouts/Code 3 Callouts/Models/Vehicles/VehicleHelper.vb
Imports Rage
Imports Stealth.Common.Models
Imports Stealth.Common.Natives

Namespace Models.Vehicles

    Public Module VehicleHelper

        Friend Function GetVehicleColors(ByVal v As Rage.Vehicle) As VehicleColor
            Return Stealth.Common.Natives.Vehicles.GetVehicleColors(v)
        End Function

        Friend Sub SetVehicleColors(ByVal v As Rage.Vehicle, ByVal pPrimColor As Integer, ByVal pSecColor As Integer)
            Functions.CallByName("SET_VEHICLE_COLOURS", pPrimColor, pSecColor)
        End Sub

    End Module

End Namespace